README - English Edits

ahandsel opened the pull request:
## Why

* Improve the README file's English
* Clarify the installation process

## What

Changes made to README:
* Capitalize Kintone's K
* Add shell or text to unspecified code blocks
* Installation Section > Add move to `/usr/local/bin` step
* Installation > Append a PATH environment variables note
* Replace curly brackets
* English review
* Table Table of Contents {New}
* Updated Installation's Step 3, 4, & Notes {New}

## How to test

* Confirm the changes on the README file locally

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/cli-kintone/blob/main/CONTRIBUTING.md)
- [x] Updated documentation if it is required.
- [x] Added tests if it is required.
- [x] Passed `yarn lint` and `yarn test` on the root directory.

### Warning from Linters
The following got flagged by the linter but were not the result of this PR:

[lint:eslint] ✖ 5 problems (0 errors, 5 warnings)

`warning ../../package.json: No license field`

`cli-kintone/src/__tests__/api.test.ts`
* 226:19  warning  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
* 251:19  warning  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap

`cli-kintone/src/kintone/client.ts`
* 65:10  warning  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap

`cli-kintone/src/record/export/usecases/__tests__/get/index.test.ts`
* 92:28  warning  Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion

`cli-kintone/src/record/import/usecases/add/record.ts`
* 84:13  warning  Blocks are nested too deeply (5). Maximum allowed is 4  max-depth

ahandsel committed:
Capitalize Kintone's K

ahandsel committed:
Add shell or text to unspecified code blocks

ahandsel committed:
Installation Section > Add move to /usr/local/bin step

ahandsel committed:
Installation > Append a PATH environment variables note

ahandsel committed:
Replace curly brackets

ahandsel committed:
English review

ahandsel committed:
Clarifying that only 1 file is needed to be extracted

ahandsel committed:
Merge branch 'kintone:main' into main

ahandsel committed:
Apply Jesslyn's suggestions from code review

Co-authored-by: jesslyn-kintone <117426935+jesslynvc@users.noreply.github.com>

ahandsel committed:
Merge pull request #1 from ahandsel/review

Review

ahandsel committed:
Merge branch 'kintone:main' into main

tasshi-me reviewed (changes requested):
Thank you for contributing a Pull Request!

I left some comments to request changes.
Could you check them, please?

ahandsel committed:
Match the Export section's Notes with Import section's

Co-authored-by: tasshi / Masaharu TASHIRO <33759872+mshrtsr@users.noreply.github.com>

ahandsel committed:
Updated Installation's Step 3, 4, & Notes

+ Removed the and moved the cli-kintone file to the /usr/local/bin bit from Step 3.
+ Added OS-specific instructions to Step 4
+ Moved the `/usr/local/bin` instruction under Notes
+ Set Installation Notes to H3

ahandsel committed:
Update Table of Contents

ahandsel committed:
Grammatical Fixes

ahandsel committed:
Updated installation notes

ahandsel committed:
Used Prettier code style

tasshi-me reviewed (approved):
LGTM 👍
